common/utils/works.js

```diff
@@ -46,12 +46,25 @@ export function getDownloadOptionsFromManifest(
       sequence => sequence['@type'] === 'sc:Sequence'
     );
   const sequenceRendering = sequence && sequence.rendering;
+  const sequenceRenderingArray = Array.isArray(sequenceRendering)
+    ? sequenceRendering
+    : [sequenceRendering];
 
-  return sequenceRendering
-    ? Array.isArray(sequenceRendering)
-      ? sequenceRendering
-      : [sequenceRendering]
+  const pdfRenderingArray = iiifManifest.mediaSequences
```

Comment:
Do we ever postfix `const`s with the type i.e. `Array`?

Reply:
Not usually, but I was doing it to be consistent with the sequenceRenderingArray const.
the sequence.rendering from the iiifManifest can either be a single object or an array. If it's the former we convert it to the latter and it's assigned to const sequenceRenderingArray, so it's clear what that is.